### DISCUSSION TITLE
Allow option of not using full classpath, but only classname

### DISCUSSION THREAD

**Discussion by larskristianhaga** [DataDog/datadog-for-intellij-platform#15] [Ideas]
When I click on a logger line in Intellij it sends me to the DataDog website, And I get this:
![Screenshot 2023-08-28 at 15 16 41](https://github.com/DataDog/datadog-for-intellij-platform/assets/13958865/4d5051f6-190c-474d-8c5a-a371bc2fdb8d)
But I get no results due to the logger only uses the Java class name instead of the full path.

If I change the query to only use the Java class name I get results as seen below:
![Screenshot 2023-08-28 at 15 16 12](https://github.com/DataDog/datadog-for-intellij-platform/assets/13958865/3a6bf1f6-700b-4bc6-ab7c-4db9a43b74e2)

I think the plugin should have an option to select/deselect to only use the Java class name instead of the full path.

Anyone else having this problem?

**Comment by abrooksv** [comment]
Hi @larskristianhaga,

Do you happen to know which logging framework you are using as well as what is the log pattern layout configuration for that framework?

**Reply by larskristianhaga** [reply]
We are using `org.slf4j:slf4j-api:1.7.2`.
And I cant seem to find a log pattern file anywhere, so I'm guessing the default - Unfortunately its a rather old and messy codebase…

**Reply by abrooksv** [reply]
Thanks for getting back to me. 

That is okay, I'll work under the assumption you are using something like `%c{1}` from [log4j2](https://logging.apache.org/log4j/2.x/manual/layouts.html#patterns) or [logback](https://logback.qos.ch/manual/layouts.html#logger) underneath of SLF4J

Can you confirm for me that you are using `org.slf4j.LoggerFactory.getLogger(Class<?> clazz)` to create the logger?

**Reply by larskristianhaga** [reply]
That is correct

**Reply by abrooksv** [reply]
Hey @larskristianhaga,

Support has been merged and will be in our 1.0.4 release
<img width="643" alt="Screenshot 2023-08-30 at 12 25 40 PM" src="https://github.com/DataDog/datadog-for-intellij-platform/assets/8992246/c646db1d-e0b6-4000-ab4b-99bba73f8389">

**Reply by larskristianhaga** [reply]
Amazing, thanks! 💯

**Reply by jfree** [reply]
Hi @larskristianhaga, 

The release is on the market place now, let us know if you have any further issues.

**Reply by larskristianhaga** [reply]
Works like a charm, thanks again
